docs/gen-ai/azure-ai-inference.md

```diff
@@ -32,7 +32,7 @@ model name is available and `{gen_ai.operation.name}` otherwise.
 
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
-| [`gen_ai.operation.name`](/docs/attributes-registry/gen-ai.md) | string | The name of the operation being performed. [1] | `chat`; `text_completion`; `embeddings` | `Required` | ![Development](https://img.shields.io/badge/-development-blue) |
+| [`gen_ai.operation.name`](/docs/attributes-registry/gen-ai.md) | string | The name of the operation being performed. [1] | `chat`; `generate_content`; `text_completion` | `Required` | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
This is good for this PR, I'll create an issue to track attribute refactoring so we won't add it to all gen ai systems

Reply:
tracking it in the https://github.com/open-telemetry/semantic-conventions/issues/1948